index.js

```diff
@@ -64,33 +71,41 @@ const expressions = {
   },
 };
 
-
-// const mapWebPacUrlToSCCURL = (url) => Object.values(expressions).find((pathtype) => {
-//   const parsedURL = new URL(url);
-//   const pathname = parsedURL.pathname
-//   const match = pathname.match(pathtype.expr);
-//   if (match) return pathtype.handler(match, parsedURL);
-//   return false;
-// });
 const mapWebPacUrlToSCCURL = (url) => {
   let redirectURL;
   for (let pathType of Object.values(expressions)) {
```

Comment:
Will these be in the same order every time? If not, might have inconsistent results (depending on the regex, of course), and might be better off with an array of objects instead. But if retrieval order is consistent then I think this is good.